test/api.postman_environment.json

```diff
@@ -0,0 +1,27 @@
+{
+	"id": "5cca63fb-26dd-4a87-8463-a55e7f56551f",
```

Comment:
Are these config files needed?  Can the config params just be passed in at runtime instead of requiring a file for each env? 
To make life easy in local testing env, could default everything to `http` and `api:3000`.